packages/fxa-settings/src/components/Settings/ProgressBar/index.tsx

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { useFtlMsgResolver } from '../../../models';
+
+export type ProgressBarProps = {
+  numberOfSteps: number;
+  currentStep: number;
+};
+
+export const ProgressBar = ({
+  numberOfSteps,
+  currentStep,
+}: ProgressBarProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedProgressBarAriaLabel = ftlMsgResolver.getMsg(
+    'progress-bar-aria-label',
+    `A progress bar showing that the user is on step ${currentStep} of ${numberOfSteps}.`,
+    { currentStep, numberOfSteps }
+  );
+  // If you have more than 6 steps, you may need to add custom widths into tailwind.
+  // https://v2.tailwindcss.com/docs/width#width-scale
```

Comment:
Nit: you're pointing to legacy tailwindcss@2 docs, but we're currently on ^3.2.0.
If I change the URL to omit the "v2." subdomain, that page doesn't seem to exist in the v3 world, but this might be closest: https://tailwindcss.com/docs/width#using-custom-values

Reply:
Ohhhhh thank you Peter!